src/main.py

```diff
@@ -0,0 +1,10 @@
+from bot.tele_bot import TeleBot 
+
+#Bot Token
+bot_token = '1006827845:AAGhqvI__6q7BYGOUwlKYLHlLpXaO-SoUYI'
+
```

Comment:
Push this to .env file and fetch all the secrets from the file . Invalidate this token and create a new one.

Reply:
Yep, rookie mistake, I will put a new token in .env

Reply:
Create new env file and push it so that I cna merge the PR